.bin/theme_changer

```diff
@@ -67,23 +67,32 @@ theme_change()
 	#bash $HOME/.bin/sudo_passhelper make -s -C ${dmenu_dir} install
 	#make -s -C ${dmenu_dir} clean
 }
+chgtk() {
```

Comment:
please change the name of the function to ```change_gtk_theme``` and move the curly brace to the next line.